randovania/games/samus_returns/logic_database/Area 3 - Factory Interior.txt

```diff
@@ -681,7 +723,9 @@ Extra - asset_id: collision_camera_029
   * Extra - actor_name: Door005
   * Extra - actor_type: doorchargecharge
   > Door to Dedicated Callisto Roost
-      Single-wall Wall Jump (Intermediate) or Climb Rooms Vertically (No High Jump)
+      Any of the following:
+          Single-wall Wall Jump (Intermediate) or Climb Rooms Vertically (No High Jump)
```

Comment:
Should revisit this since IBJ would need to deal with the water. Either adjust it like the A2 Alpha NW escape or leave a TODO